Hunted Mobile/Hunted Mobile/Hunted Mobile/Repository/InviteKeyRepository.cs

```diff
@@ -38,6 +38,7 @@ public class InviteKeyRepository {
                 }
             }
             catch(Exception e) {
+                Console.WriteLine(e.ToString());
```

Comment:
Dit was toch ook juist vanuit Stijn gekomen dat we geen `Console.WriteLine` in onze applicatie willen hebben?  Misschien dan de `Exception` gewoon weglaten.

Reply:
`Console.WriteLine` is zeker niet optimaal, maar het lijkt me iets beter dan een leeg catch block. Met een lege catch weet je namelijk niks als het fout gaat en is het moeilijk om erachter te komen waarom iets fout gaat.
`Console.WriteLine` met debug informatie moet wel weggehaald worden.